lua/entities/gmod_wire_expression2/core/egpfunctions.lua

```diff
@@ -934,6 +934,21 @@ e2function array wirelink:egpVertices( number index )
 	return {}
 end
 
+--------------------------------------------------------
+-- Indexes
+--------------------------------------------------------
+__e2setcost(1)
+e2function array wirelink:egpObjectIndexes()
+	if not EGP:ValidEGP(this) then return {} end
+	if not this.RenderTable or #this.RenderTable == 0 then return {} end
+	local indexes = {}
+	for k, v in pairs(this.RenderTable) do
+		indexes[#indexes + 1] = v.index
+		self.prf = self.prf + 1/3
```

Comment:
self.prf = self.prf + #indexes/3

Reply:
Doing it after would not increment it while it is still iterating. So, if you have a lot of objects, it would error out while still processing/iterating the table.
I consider this to be better than doing it before/after using the length (that what you said).
But, if you really want it work that way, fine.

Reply:
That's not how `prf` works, it's only checked at the beginning of `seq` statements and after code execution.

Reply:
@bigdogmat Alright then, thanks for that.